addons/knowledge/static/src/js/knowledge_renderers.js

```diff
@@ -256,6 +233,22 @@ const KnowledgeFormRenderer = FormRenderer.extend({
         $container.prepend(items);
     },
 
+    /**
+     * Renders the emoji picker
+     */
+    _renderEmojiPicker: function () {
```

Comment:
maybe there should be a way to instantiate only one emoji picker, and trigger an "Open emoji Picker" on icon click.
Because instantiate 234 emoji picker because there are 234 article in the sidebar seems a bit overkill

Reply:
Ok, I found a trick to avoid instantiating the widget on start up: The idea is to attach a listener on each dropdown item and instantiate the widget dynamically when the user clicks on it. With the `one` function of JQuery, the listener will be detached after the first execution. So, there will be at most one widget per article.